lib/LedgerSMB/Scripts/import_csv.pm

```diff
@@ -65,6 +65,61 @@ our $cols = {
                          bom partsgroup / ],
 };
 
+my $header_info = {
+    gl => {
+        title => 'Journal Import',
+        info => ''
```

Comment:
```perl
info => "Imports a CSV file with the following columns (in exactly the order specified): @{$cols->{gl}}"
```

There's one problem with this approach: we can't mark these strings for translation when we use the interpolation as shown.

Or we'd need:
```perl
use LedgerSMB::I18N;

...

info => LedgerSMB::I18N::text('Imports a CSV file with the following columns (in exactly the order specified): [_1]', join(' ', @{$cols->{gl}}))
```

Reply:
I tried to use LedgerSMB:I18N as you said in import_csv.pm. I am not able to start the container with following error.
```
Uncaught exception from user code:

Error while loading /srv/ledgersmb/bin/ledgersmb-server.psgi: Can't locate object method "Locale" via package "LedgerSMB::App_State" at /srv/ledgersmb/lib/LedgerSMB/I18N.pm line 100.

Compilation failed in require at /usr/share/perl5/Module/Runtime.pm line 314.

Plack::Util::load_psgi("/srv/ledgersmb/bin/ledgersmb-server.psgi") called at /usr/share/perl5/Plack/Runner.pm line 181

Plack::Runner::__ANON__() called at /usr/share/perl5/Plack/Runner.pm line 24

Plack::Runner::__ANON__() called at /usr/share/perl5/Plack/Runner.pm line 24

Plack::Runner::__ANON__() called at /usr/share/perl5/Plack/Runner.pm line 24

Plack::Runner::__ANON__() called at /usr/share/perl5/Plack/Runner.pm line 24

Plack::Runner::__ANON__() called at /usr/share/perl5/Plack/Loader.pm line 55

Plack::Loader::preload_app(Plack::Loader=HASH(0x55bd1acfd530), CODE(0x55bd1af301e0)) called at /usr/share/perl5/Plack/Runner.pm line 276

Plack::Runner::run(Plack::Runner=HASH(0x55bd1aa6b8e0)) called at /usr/bin/plackup line 7
```

Do you have any clue why it happens?

Reply:
oh. ok. I know what the problem is: this code needs to run inside a `sub` (the code calling `*::text()`).

Reply:
Thank you, I got it. So shall I commit the changes or shall we stop the PR here and move to vue?

Do you want this changes to back port to 1.9?

Reply:
I don't think we need to backport to 1.9: it's not fixing an immediate problem although it does address a major usability problem. Normally, I'd expect that to go into 1.10, whereas for bugfixes, I'd expect us to be able to create a change which allows backporting.

The good part of moving this to Vue is that it raises a number of important questions which we need to solve when we want to continue with Vue: E.g. where/how do we get the (translated!) content (which we call 'info variable' in this PR).